lib/config/utils.js

```diff
@@ -20,3 +23,23 @@ exports.toIntegerConfig = function toIntegerConfig (configValue) {
   }
   return configValue
 }
+
+exports.getGitCommit = function getGitCommit (repodir) {
+  if (!fs.existsSync(repodir + '/.git/HEAD')) {
+    return undefined
+  }
+  let reference = fs.readFileSync(repodir + '.git/HEAD', 'utf8')
```

Comment:
A slash is required before `'.git/HEAD'` because `repodir` may not be ended with a slash.

Reply:
Oh I missed that one ^^